stl/inc/generator

```diff
@@ -409,25 +412,27 @@ private:
 };
 
 _EXPORT_STD template <class _Rty, class _Vty, class _Alloc>
-class generator : public ranges::view_interface<generator<_Rty, _Vty, _Alloc>> {
+class generator : public _RANGES view_interface<generator<_Rty, _Vty, _Alloc>> {

```

Comment:
Why use `_RANGES` macro on types? Are they susceptible to ADL?

Reply:
We're in `std`, not `std::ranges` here, so we need to qualify. Qualifying with `_RANGES` consistently makes searching easier.